src/Bellatrix.Web/services/BrowserService.cs

```diff
@@ -214,6 +214,25 @@ public void WaitForAjax()
             onTimeout: () => { Logger.LogWarning($"Timed out waiting for open connections to be closed. Wait time: {maxSeconds} sec."); });
     }
 
+    public void WaitForIframeRequest(string requestPartialUrl, int additionalTimeoutInSeconds = 0)
+    {
```

Comment:
if I remember correctly you told me that you found a way to do it with WebDriverWait?

Reply:
Yes, we can close the current PR, because it seems the following method works fine.
`Wait.Until(() => ExpectedConditions.FrameToBeAvailableAndSwitchToIt(Frame.LocatorValue) is not null);`